docs/tutorials/basic_ua_commands.md

```diff
@@ -0,0 +1,258 @@
+$ sudo ua refresh
+```
+
+This command will refresh the contract on your machine. This command is also really useful if you
+want to change any definitions on your subscription. For example, let's assume that you now want
+`cis` to be enabled by default when attaching. After you modify your subscription for that, running
+the refresh command will process any changes that were performed in the subscription, enabling
+`cis` because of this.
+
+### Enable
+
+There is another way to enable a service that wasn't activated during attach or refresh.
+Suppose that you don't want to modify your subscription, but still want to enable `cis`
+on the machine. To achieve that, You can use the enable command.
+
+Let's enable `cis` on our LXD container by running:
+
+```console
+$ sudo ua enable cis
+```
+
+After running it, you should see an output similar to this one:
+
+```
+One moment, checking your subscription first
+Updating package lists
+Installing CIS Audit packages
+CIS Audit enabled
+Visit https://security-certs.docs.ubuntu.com/en/cis to learn how to use CIS
+```
+
+You can confirm that `cis` is enabled now by running:
+
+```console
+$ ua status
+```
+
+And you should see:
+```
+SERVICE       ENTITLED  STATUS    DESCRIPTION
+cis           yes       enabled   Center for Internet Security Audit Tools
+esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
+fips          yes       n/a       NIST-certified core packages
+fips-updates  yes       n/a       NIST-certified core packages with priority security updates
+livepatch     yes       n/a       Canonical Livepatch service
+```
+
+You can see now that `cis` is marked as `enabled` on status.
+
+
+### Disable
+
+Let's suppose that you don't want a service anymore, you can also disable any service offering
+through UA. For example, let's disable the `cis` service you just enabled by running on the LXD
+container:
+
+```console
+$ sudo ua disable cis
+```
+
+After running that command, let's now run status to see what happened to `cis`:
+```
+SERVICE       ENTITLED  STATUS    DESCRIPTION
+cis           yes       disabled  Center for Internet Security Audit Tools
+esm-apps      yes       enabled   UA Apps: Extended Security Maintenance (ESM)
+esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
+fips          yes       n/a       NIST-certified core packages
+fips-updates  yes       n/a       NIST-certified core packages with priority security updates
+livepatch     yes       n/a       Canonical Livepatch service
+```
+
+You can see that `cis` status is back to disabled.
+
+```
+Note: the disable command doesn't uninstall any package that was installed by
+the service. The command only removes the access you have to the service, but it
+doesn't undo any configuration that was applied on the machine.
+```
+
+### Detach
+
+Finally, what if I don't want to be attached to an UA subscription any longer ?
+You can achieve that through the `detach` command:
```

Comment:
I haven't paid close enough attention to this, but we should stick with one "subject" throughout the tutorial. In other words, we should pick one of "I", "We", or "You" and use it consistently. I think we've mostly been using "We", so lets keep that going here.

Also some minor things

```suggestion
Finally, what if we don't want this machine to be attached to a UA subscription any longer?
We can achieve that using the `detach` command:
```

Reply:
That's a good point. I am trying to use `you` in this doc. The places were I am using `we` are places were I mentioning that the UA team will perform an action. But let me know if  ẁe` is better instead of `you` on the docs

Reply:
I somehow didn't see @renanrodrigo's comment that says pretty much the exact same thing :facepalm: 

I think `you` is good too. And `we` would make sense in those situation :+1: